app/services/iugu.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
Should I already add specs for this guy? It's already covered by the others.

Reply:
Not yet. If it is covered, it's OK ;)